### DISCUSSION TITLE
Why Selfhosted Version Different Than The Local Version?

### DISCUSSION THREAD

**Discussion by weir1** [toeverything/AFFiNE#3962] [Q&A]
I first checked demo & not impressed, But I dont Know I needed a solution desperately so I tried on my windows & super amazed then I SSH into my server & installed via docker but Its nowhere near to experience I am getting on my windows?
How to install on my server the same as I have installed on my windows

**Comment by himself65** [comment]
Self host version are under heavy development and it has a long time no update, please use our desktop version for now.

**Comment by figadore** [comment]
> Self host version are under heavy development and it has a long time no update, please use our desktop version for now.

Is there an issue I can follow for updates? 

Anything a new contributor can do to help?

**Comment by HeJiachen-PM** [comment]
No worries, we are finally releasing a new selfhost version. And yes, you
can always check the "first good issue".

Reese ***@***.***>于2023年10月25日 周三13:52写道：

**Reply by mtojay** [reply]
nice. cant wait. hope itll release soon

**Reply by Julian-Hackenberg** [reply]
Any updates on this?

**Reply by EYHN** [reply]
We can follow the updates in https://github.com/toeverything/AFFiNE/issues/5069
